### PR TITLE
#1149 Tail indexes

### PR DESCRIPTION
**Pull request by uweschaefer**

### CODE REVIEW

File: factcast-server-grpc/src/main/java/org/factcast/server/grpc/FactStoreGrpcService.java [hunk 1]
```diff
@@ -147,11 +163,12 @@ public void subscribe(
             new BlockingStreamObserver<>(
                 req.toString(), (ServerCallStreamObserver) responseObserver);
 
+        long startingAfterSerial =
```
Comment (otbe) [inline factcast-server-grpc/src/main/java/org/factcast/server/grpc/FactStoreGrpcService.java:166]: unusded?

File: factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/internal/tail/PGTailIndexManager.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package org.factcast.store.pgsql.internal.tail;
+
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock;
+import org.factcast.core.subscription.observer.FastForwardTarget;
+import org.springframework.scheduling.annotation.Scheduled;
+
+public interface PGTailIndexManager extends FastForwardTarget {
+  @Scheduled(cron = "* * */1 * * *")
```
Comment (otbe) [inline factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/internal/tail/PGTailIndexManager.java:8]: Should this stay here? 🤔

File: factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/internal/tail/PGTailIndexingConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import org.factcast.store.pgsql.PgConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+@Configuration
+public class PGTailIndexingConfiguration {
+
+  @Bean
+  public PGTailIndexManagerImpl pgTailIndexManager(
```
Comment (otbe) [inline factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/internal/tail/PGTailIndexingConfiguration.java:12]: Maybe use the interface here?

File: factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/internal/PgFactStore.java [hunk 4]
```diff
@@ -80,20 +82,23 @@ public PgFactStore(
...
     this.lock = lock;
+    this.pgFactIdToSerialMapper = pgFactIdToSerialMapper;
     this.snapCache = snapCache;
     this.metrics = metrics;
     this.factTransformerService = factTransformerService;
   }
 
   @Override
   public @NonNull Optional<Fact> fetchById(@NonNull UUID id) {
+    // TODO lets see if this approach is the fastest
```
Comment (otbe) [inline factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/internal/PgFactStore.java:101]: It was already changed in another PR wasnt it? 🤔

Comment (otbe) [inline factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/internal/PgFactStore.java:101]: It was already changed in another PR wasnt it? 🤔

File: factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/internal/tail/PGTailIndexManager.java (hunk 2 above)
Comment (uweschaefer) [reply factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/internal/tail/PGTailIndexManager.java:8]: niet

File: factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/internal/PgFactStore.java (hunk 4 above)
Comment (uweschaefer) [reply factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/internal/PgFactStore.java:101]: yes, but not on master yet

**Pull request merged**